BuildResidentialHPXML/measure.rb

```diff
@@ -3388,23 +3363,58 @@ def self.create(runner, model, args, epw_path)
     end
 
     # Check for errors in the HPXML object
-    if args[:apply_validation].is_initialized && args[:apply_validation].get
+    if args[:apply_validation]
       errors = hpxml.check_for_errors()
       if errors.size > 0
         fail "ERROR: Invalid HPXML object produced.\n#{errors}"
       end
     end
```

Comment:
Can we move this into the `validate_hpxml` method? I don't think it needs to be handled separately here.